### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Update vulnerabilities flagged by dependabot 

## Motivation and Context
Fix vulnerabilities 

## How Has This Been Tested?
Confirmed updated dependency version and Apps build succeeds https://github.com/Rise-Vision/rise-vision-apps/pull/2159

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good, thanks!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "widget-tester",
-  "version": "1.11.4",
+  "version": "1.12.0",
```
Comment (alex-deaconu) [inline package.json:3]: @ezequielc curious why bump to `12` instead of `11.5`? Thanks.

Comment (ezequielc) [reply package.json:3]: @alex-deaconu Mainly to be extra-cautious given the error we saw in phantomJS with the regular update, before setting the fixed mocha version. Let me know if you think we should change to `11.5`. Thanks

Comment (alex-deaconu) [reply package.json:3]: @ezequielc nope, let's leave as is. Thanks.

**Pull request merged**